shell/common/input_events_unittests.cc

```diff
@@ -224,53 +224,56 @@ TEST_F(ShellTest, DelayAtMostOneEventForFasterThanVSyncInputEvents) {
-  constexpr double iphone_xs_times[] = {0.15,
-                                        1.0773046874999999,
-                                        2.1738720703124996,
-                                        3.0579052734374996,
-                                        4.0890087890624995,
-                                        5.0952685546875,
-                                        6.1251708984375,
-                                        7.1253076171875,
-                                        8.125927734374999,
-                                        9.37248046875,
-                                        10.133950195312499,
-                                        11.161201171875,
-                                        12.226992187499999,
-                                        13.1443798828125,
-                                        14.440327148437499,
-                                        15.091684570312498,
-                                        16.138681640625,
-                                        17.126469726562497,
-                                        18.1592431640625,
-                                        19.371372070312496,
-                                        20.033774414062496,
-                                        21.021782226562497,
-                                        22.070053710937497,
-                                        23.325541992187496,
-                                        24.119648437499997,
-                                        25.084262695312496,
-                                        26.077866210937497,
-                                        27.036547851562496,
-                                        28.035073242187497,
-                                        29.081411132812498,
-                                        30.066064453124998,
-                                        31.089360351562497,
-                                        32.086142578125,
-                                        33.4618798828125,
-                                        34.14697265624999,
-                                        35.0513525390625,
-                                        36.136025390624994,
-                                        37.1618408203125,
-                                        38.144472656249995,
-                                        39.201123046875,
-                                        40.4339501953125,
-                                        41.1552099609375,
-                                        42.102128906249995,
-                                        43.0426318359375,
-                                        44.070131835937495,
-                                        45.08862304687499,
-                                        46.091469726562494};
+  //
+  // We don't use `constexpr double iphone_xs_times` here because Fuchsia's
+  // clang doesn't handle it well with lambda capture.
+  double iphone_xs_times[] = {0.15,
```

Comment:
Can this not be `const`?

Reply:
Not sure. Let's see whether the compiler complains :)

Reply:
seems fine, switched to const